index.html

```diff
@@ -14,7 +14,8 @@ <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
-      <nav id="desktop-nav">
+      <nav id="desktop-nav" class="nav">
```

Comment:
![code4](https://user-images.githubusercontent.com/53111464/132109371-bd6d0d28-1ece-4a2c-a199-2f03e412ecbf.png)

- [ ] In mobile devices when any of the links are clicked. The links in the navbar have to disappear.